release-notes/versioned/pulsar-2.11.1.md

```diff
@@ -0,0 +1,190 @@
+- [Java] Fix DeadLetterProducer creation callback blocking client io thread [#19930](https://github.com/apache/pulsar/pull/19930)
+
+### Pulsar IO and Pulsar Functions
+- [Functions] Ensure prometheus metrics are grouped by type [#15558](https://github.com/apache/pulsar/pull/15558)
+- [Functions] Fix k8s merge runtime opts bug [#19481](https://github.com/apache/pulsar/pull/19481)
+- [Functions] Use AuthorizationServer more in Function Worker API [#19975](https://github.com/apache/pulsar/pull/19975)
+- [IO Connector] Debezium sources: Support loading config from secrets [#19205](https://github.com/apache/pulsar/pull/19205)
+- [IO Connector] Elasticsearch sink: Support loading config from secrets [#19206](https://github.com/apache/pulsar/pull/19206)
+- [IO Connector] Update Elasticsearch sink idle cnx timeout to 30s [#19377](https://github.com/apache/pulsar/pull/19377)
+- [IO Connector] DBC sink: allow any jdbc driver [#17951](https://github.com/apache/pulsar/pull/17951)
+- [IO Connector] ElasticSearch sink: align null fields behaviour [#18577](https://github.com/apache/pulsar/pull/18577)
+- [IO Connector] KCA sink: handle null values with KeyValue<Avro,Avro> schema [#19861](https://github.com/apache/pulsar/pull/19861)
+- [IO Connector] KCA: option to collapse partitioned topics [#19923](https://github.com/apache/pulsar/pull/19923)
+- [IO Connector] KCA: flag to force optional primitive schemas [#19951](https://github.com/apache/pulsar/pull/19951)
+-
+
+### Observability
+- [Broker] Fix wrong metrics text generated when label_cluster specified [#17704](https://github.com/apache/pulsar/pull/17704)
+- [Broker] Add metrics for transaction [#15140](https://github.com/apache/pulsar/pull/15140)
+- [Broker] Remove timestamp from Promtheus metrics [#17419](https://github.com/apache/pulsar/pull/17419)
+- [Broker] Fix incorrect bundle split count metric [#17970](https://github.com/apache/pulsar/pull/17970)
+- [Broker] Update authentication failed metrics report [#17787](https://github.com/apache/pulsar/pull/17787)
+- [Function] Fix invalid metric type "gauge " [#18129](https://github.com/apache/pulsar/pull/18129)
+
+### CLI
+- [Pulsar Admin] Returns 4xx error when pulsar-worker-service is disabled and trying to access it [#17901](https://github.com/apache/pulsar/pull/17901)
+- [Pulsar Admin] PulsarAdminBuilderImpl overrides timeout properties passed through config map [#17375](https://github.com/apache/pulsar/pull/17375)
+- [Pulsar Admin] Improve partitioned-topic condition evaluation [#19015](https://github.com/apache/pulsar/pull/19015)
+- [Pulsar Admin] Make the default value of param --get-subscription-backlog-size of admin API topics stats true [#19302](https://github.com/apache/pulsar/pull/19302)
+- [Pulsar Admin] Make response code to 400 instead of 500 when delete topic fails due to enabled geo-replication [#19879](https://github.com/apache/pulsar/pull/19879)
```

Comment:
```suggestion
- [Pulsar Admin] Make response code to 400 instead of 500 when deleting topic fails due to enabled geo-replication [#19879](https://github.com/apache/pulsar/pull/19879)
```

Reply:
fixed